packages/entities/entities-shared/src/composables/useFetchUrlBuilder.ts

```diff
@@ -7,15 +7,15 @@ import type { MaybeRef } from '../types/utils'
 import type { TableDataFetcherParams } from '@kong/kongponents'
 
 export default function useFetchUrlBuilder(
-  config: MaybeRef<KonnectBaseTableConfig | KongManagerBaseTableConfig>,
+  config: MaybeRefOrGetter<KonnectBaseTableConfig | KongManagerBaseTableConfig>,
   baseUrl: MaybeRef<string>,
 ) {
```

Comment:
@Leopoldthecoder, I saw the `baseUrl` was marked to `MaybeRef` as well, maybe it is better if we pass a ref as well?

Reply:
Updated.